docs/MoodleApp.md

```diff
@@ -0,0 +1,20 @@
+---
+layout: page
+title: Moodle App
+---
+
+In order to test plugins with mobile support, the only command that needs a special configuration is `behat`.
+
+In practice, you only need to set the `MOODLE_APP` env variable to `true`, and all the dependencies will be configured during the `install` command. You can also configure the behaviour of [Acceptance testing for the Moodle App](https://moodledev.io/general/app/development/testing/acceptance-testing) using the following env variables:
+
+- `MOODLE_APP_DOCKER_IMAGE`: Tag of [the Moodle App Docker image](https://moodledev.io/general/app/development/setup/docker-images) to use for running the app. The default value is `moodlehq/moodleapp:latest-test`.
+- `MOODLE_APP_BEHAT_PLUGIN_PROJECT`: Project in github to use for installing the plugin with Behat steps specific to the Moodle App. The default value is `moodlehq/moodle-local_moodleappbehat`. This variable will be ignored if `MOODLE_APP_BEHAT_PLUGIN_REPOSITORY` is set.
+- `MOODLE_APP_BEHAT_PLUGIN_REPOSITORY`: Repository url to use for installing the plugin with Behat steps specific to the Moodle App. By default, the github repository defined in `MOODLE_APP_BEHAT_PLUGIN_PROJECT` will be used.
```

Comment:
Why do we need this duality? Sure there is a reason, but I cannot get it now.

Reply:
The first one, `MOODLE_APP_BEHAT_PLUGIN_PROJECT`, is used if the plugin is hosted on github so you can just set it as `moodlehq/moodle-local_moodleappbehat`; whilst the second one, `MOODLE_APP_BEHAT_PLUGIN_REPOSITORY`, takes a full url so you can use repositories that aren't hosted in github, like `https://gitlab.com/moodlehq/moodle-local_moodleappbehat.git` (this url doesn't exist, just illustrating the idea).

This is actually the same that happens with the `--project` and `--clone` arguments in [the add-plugin command](https://github.com/moodlehq/moodle-plugin-ci/blob/main/src/Command/AddPluginCommand.php), that's why I did it. I guess I can change the naming or documentation if you want. But I think "repository" is a better name than "clone" for an env variable.

Reply:
Ok, clear. Resolving.

Reply:
Maybe we can consider renaming the `--clone` option (while keeping it working for some good time) and use `--repository` there. I've created #260 about it.